src/questions/funcTypeQuestions.tsx

```diff
@@ -6,6 +6,7 @@ export interface Question {
   prompt: string | JSX.Element;
   answers: string[];
   type: QuestionType;
+  ID?: number;
```

Comment:
nit: make ID lowercase

Reply:
also, what are we using this for atm? doesn't seem like it's being used anywhere

Reply:
This wasn't add in yet, but when we want to randomize the question order, the id will be used to determine which question is which